uk.et1.big-finish-downloader.yaml

```diff
@@ -0,0 +1,30 @@
+id: uk.et1.big-finish-downloader
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: big-finish-downloader-gtk
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --filesystem=home
```

Comment:
What does it needs home access for?

Reply:
To write the files it downloads to the user's home directory

Reply:
I'd rather change it to `xdg-music, `xdg-download`, `/media` and `/run/media`, but it's ultimately your decision.